packages/trader/src/Modules/Trading/Components/Form/Purchase/contract-info.jsx

```diff
@@ -76,18 +76,6 @@ const ContractInfo = ({
     const { message, obj_contract_basis, stake } = proposal_info;
 
     const setHintMessage = () => {
-        if (is_turbos) {
-            return (
-                <Localize
-                    i18n_default_text='<0>{{title}}</0> {{message}}'
```

Comment:
@henry-deriv I think we still need to show the message, isn't? What I meant was to remove localize from here since it's not needed 😄